dashboard/db/migrate/20160617163547_create_user_geos.rb

```diff
@@ -0,0 +1,16 @@
+class CreateUserGeos < ActiveRecord::Migration
+  def change
+    create_table :user_geos do |t|
+      t.references :user, index: true, unique: true, foreign_key: true, null: false
+      t.integer :user_id
+      t.timestamps null: false
+      t.datetime :indexed_at, null: false
+      t.string :ip_address
+      t.string :city
+      t.string :state
+      t.string :country
+      t.string :postal_code
+      t.string :latitude_longitude
```

Comment:
Much as I like consistency, I can't see this not complicating future implementations